corehq/toggles/__init__.py

```diff
@@ -2529,3 +2520,14 @@ def domain_has_privilege_from_toggle(privilege_slug, domain):
     # TODO: Move to public wiki
     help_link='https://confluence.dimagi.com/display/saas/Show+Application+Error+Report+Feature+Flag'
 )
+
+DATA_DICTIONARY = FrozenPrivilegeToggle(
+    privileges.DATA_DICTIONARY,
+    'data_dictionary',
+    label='Project level data dictionary of cases',
+    tag=TAG_SOLUTIONS_OPEN,
+    namespaces=[NAMESPACE_DOMAIN],
+    description='Project level data dictionary of cases',
+    # TODO: Move to public wiki
```

Comment:
Is this PR waiting on the public docs to be updated or will this be merged regardless?

Reply:
Given the usage of this feature, I think we should wait until the relevant documentation has been updated. I'll then be able to update the help link in this PR and the documentation will be available to users once this change goes live.